src/utils/contractHelpers.ts

```diff
@@ -125,6 +126,10 @@ export const getClaimRefundContract = (signer?: ethers.Signer | ethers.providers
 export const getTradingCompetitionContract = (signer?: ethers.Signer | ethers.providers.Provider) => {
   return getContract(tradingCompetitionAbi, getTradingCompetitionAddress(), signer)
 }
+
+export const getTradingCompetitionContractV2 = (signer?: ethers.Signer | ethers.providers.Provider) => {
+  return getContract(tradingCompetitionV2Abi, getTradingCompetitionAddress(), signer)
```

Comment:
Should probably make `getTradingCompetitionV2Address()`. I.e. let's not overwrite 1st competition address.
We can keep the all the 1st trading competition things in the codebase since we'd probably have some kind of "competition history" page in the future.